.devcontainer/devcontainer.json

```diff
@@ -23,7 +23,13 @@
 		"PS_VERSION": "7.2.7"
 	  }
 	},
-	"runArgs": ["--cap-add=SYS_PTRACE", "--security-opt", "seccomp=unconfined"],
+	"runArgs": ["--cap-add=SYS_PTRACE", 
```

Comment:
Why remove `seccomp=unconfined`?

Reply:
I didn't actually remove it, I just changed the formatting:

```json
"runArgs": ["--cap-add=SYS_PTRACE", 
				"--security-opt", 
				"seccomp=unconfined"
			]
```